core/src/main/java/tachyon/Constants.java

```diff
@@ -59,6 +59,9 @@
   public static final int DEFAULT_WORKER_MAX_WORKER_THREADS = 2048;
 
   public static final int DEFAULT_USER_FAILED_SPACE_REQUEST_LIMITS = 3;
+  
+  public static final boolean DEFAULT_USER_DISABLE_LOCAL_READ = false;
```

Comment:
config param is usually about enable xxx, how about: 
DEFAULT_USER_ENABLE_LOCAL_READ = true?